packages/superset-ui-chart-controls/package.json

```diff
@@ -26,14 +26,14 @@
     "access": "public"
   },
   "dependencies": {
+    "@react-icons/all-files": "^4.1.0",
     "@superset-ui/core": "0.17.64",
     "lodash": "^4.17.15",
     "prop-types": "^15.7.2"
   },
   "peerDependencies": {
     "@types/react": "*",
     "antd": "^4.9.4",
-    "react": "^16.13.1",
-    "react-icons": "^4.2.0"
```

Comment:
note: moved from being a peer dep because i'm going to remove it from superset proper (it's not being used anywhere there)